src/app/pages/import-wallet/step1/step1.component.tsx

```diff
@@ -24,9 +24,14 @@ export class ImportWalletStep1 extends Component<IProps, IState> {
             words: [],
             error: false
         };
+
+        this.onWordsInputChange = this.onWordsInputChange.bind(this);
```

Comment:
you can use the `@bind` decorator instead of this. I found it recently so I didn't refactor all the places where  `.bind()` is used.